docs/environment.rst

```diff
@@ -101,15 +101,14 @@ Wi-Fi SSID to auto-connect to even if user code is not running.
 Additional board specific keys
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-`MaTouch ESP32-S3 Parallel TFT with Touch 7“ <https://circuitpython.org/board/makerfabs_tft7/>`_
-
-CIRCUITPY_DISPLAY_WIDTH
-~~~~~~~~~~~~~~~~~~~~~~~
+CIRCUITPY_DISPLAY_WIDTH (Sunton, MaTouch)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Selects the correct screen resolution (1024x600 or 800x640) for the particular board variant.
-If the CIRCUITPY_DISPLAY_WIDTH parameter is set to a value of 1024 the display is initialized
+If the CIRCUITPY_DISPLAY_HEIGHT parameter is set to a value of 1024 the display is initialized
```

Comment:
I don't think you want this changed. 1024 is the width afaict.

Reply:
done